City-Bureau/city-scrapers
# 🚫 Deprecate spider: Commission on Chicago Landmarks

## What's this PR do?

Deletes our Commission on Chicago Landmarks spider (aka. `chi_landmark_commission`).

## Why are we doing this?

The agency's website has changed significantly and the spider is now broken. Based on internal discussion, it's not worth rebuilding this particular spider.

## Steps to manually test

After installing the project using `pipenv`:

1. Activate the virtual environment:
```
pipenv shell
```
2. Run the spider:
```
scrapy crawl chi_landmark_commission -O test_output.csv
```
3. Monitor the stdout and ensure that the crawl proceeds without raising any errors. Pay attention to the final status report from scrapy.

4. Inspect `test_output.csv` to ensure the data looks valid. I suggest opening a few of the URLs under the source column of test_output.csv and comparing the data for the row with what you see on the page.

## Are there any smells or added technical debt to note?
